iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -7,14 +7,24 @@
 namespace Microsoft.Azure.Devices.Common.Exceptions
 {
     /// <summary>
-    /// Error Codes for common IoT hub exceptions.
+    /// Error codes for common IoT hub exceptions.
     /// </summary>
     public enum ErrorCode
     {
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Do we need to suppress this after adding the xml comments?

Reply:
When we finish, I'll remove the suppression.

Reply:
Note this is for the whole file, as the end pragma is at the end of the class.